README.md

```diff
@@ -59,10 +59,10 @@ Optional vars:
 
   | Var      | Description | Default |
   | -------- | ----------- | ------- |
-  | services_instance_type | Instance type for the centralized services box.  We recommend a m4 instance | m4.2xlarge |
-  | builder_instance_type | Instance type for the 1.0 builder machines.  We recommend a r3 instance | r3.2xlarge |
+  | services_instance_type | Instance type for the centralized services box.  We recommend a m4 instance | m5.2xlarge |
+  | builder_instance_type | Instance type for the 1.0 builder machines.  We recommend a r3 instance | r5.2xlarge |
```

Comment:
Minor: the recommended doesn't match what we are defaulting to.

Reply:
Pushed some additional commits to address this.  Let me know if the changes resolve the issue please.